### PR TITLE
feat(account.tracking): add tracking in account section

### PR DESCRIPTION
**Pull request by marie-j**
### Needs 
- [X] https://github.com/ovh-ux/ovh-ui-angular/pull/306 ( ⚠️ waiting for release)

### Related 
- [ ] https://github.com/ovh-ux/ovh-angular-otrs/pull/29

MBE-137

### CODE REVIEW

File: client/app/account/user/ssh/user-ssh.html [hunk 1]
```diff
@@ -15,46 +15,45 @@ <h1 data-translate="user_ssh_title"></h1>
             <p>
                 <strong data-translate="user_ssh_key_explain_1"></strong>
             </p>
-            <p data-ng-bind-html="'user_ssh_key_explain_2' | translate:{ t0: ctrlSsh.guides.sshCreate }"></p>
+            <p>
+                <span data-translate="user_ssh_key_explain_2"></span>
+                <span data-translate="user_ssh_key_explain_2_guide"
+                      data-translate-values="{ t0: ctrlSsh.guides.sshCreate }"
+                      data-translate-compile></span>
```
Comment (marie-j) [inline client/app/account/user/ssh/user-ssh.html]: To allow tracking directive to use current scope and not `angular-translate`'s one

File: client/app/account/billing/autoRenew/billing-autoRenew.html [hunk 2]
```diff
@@ -108,44 +108,44 @@ <h1 data-translate="autorenew_title"></h1>
...
-                </div>
+                <oui-dropdown class="d-inline-block">
+                    <oui-dropdown-trigger data-text="{{:: 'autorenew_service_action_title' | translate }}"></oui-dropdown-trigger>
+                    <oui-dropdown-content>
+                        <a class="oui-dropdown-option"
+                           data-ui-sref="app.account.billing.payment.mean"
+                           data-track-on="click"
+                           data-track-type="action"
+                           data-track-name="services::payment_types::edit::payment_types_settings">
+                           <span data-translate="autorenew_manage_my_payment_means"></span>
```
Comment (JeremyDec) [inline client/app/account/billing/autoRenew/billing-autoRenew.html]: Missing one space :)

File: client/app/account/billing/autoRenew/billing-autoRenew.html [hunk 3]
```diff
@@ -108,44 +108,44 @@ <h1 data-translate="autorenew_title"></h1>
...
+                        </a>
+                        <a data-ng-if="automaticRenewV2Mean.available"
+                           class="oui-dropdown-option"
+                           target="_blank"
+                           rel="noopener"
+                           data-ng-href="{{ urls.renewAlign }}"
+                           data-track-on="click"
+                           data-track-type="action"
+                           data-track-name="services::autoRenew::edit::autoRenew_date_settings">
+                           <span data-translate="autorenew_service_update_date_button_caption"></span>
```
Comment (JeremyDec) [inline client/app/account/billing/autoRenew/billing-autoRenew.html]: Missing one space :)

File: client/app/account/billing/mean/billing-mean.html [hunk 4]
```diff
@@ -7,24 +7,29 @@
...
-        <a class="btn btn-default"
+        <a class="oui-button oui-button_secondary"
            data-ng-if="canAddPaymentMeans"
-           data-ng-href="#/billing/mean/add"
-           data-translate="payment_mean_add_title">
+           data-ui-sref="app.account.billing.payment.meanAdd"
+           data-track-on="click"
+           data-track-type="action"
+           data-track-name="payment_types::payment_types::add::add_payment_types">
+           <span data-translate="payment_mean_add_title"></span>
```
Comment (JeremyDec) [inline client/app/account/billing/mean/billing-mean.html]: Missing one space :)

File: client/app/account/billing/orders/billing-orders.html [hunk 5]
```diff
@@ -131,60 +131,48 @@ <h1 data-translate="orders_page_title"></h1>
...
+                               rel="noopener"
+                               class="oui-dropdown-option"
+                               data-ng-if="order.url && !order.bill"
+                               data-ng-href="{{ ::order.url }}"
+                               data-ng-attr-title="{{ 'table_download_html_order' | translate }}"
+                               data-track-on="click"
+                               data-track-type="action"
+                               data-track-name="orders::open_invoices">
+                               <span data-translate="table_download_html_order"></span>
+                               <span class="oui-icon oui-icon-page_line" aria-hidden="true"></span>
```
Comment (JeremyDec) [inline client/app/account/billing/orders/billing-orders.html]: Same

File: client/app/account/user/agreements/user-agreements.html [hunk 6]
```diff
@@ -151,13 +151,15 @@ <h2 data-translate="user_agreements_list_title"></h2>
...
                            data-ng-href="{{agreement.pdfUrl}}"
-                           title="{{ 'user_agreements_download_pdf_title' | translate:{ t0: agreement.name } }}">
-                            <span class="fa fa-download"
-                                  aria-hidden="true">
-                            </span>
+                           data-track-on="click"
+                           data-track-type="action"
+                           data-track-name="contracts::download::download_contracts">
+                           <span class="sr-only" data-translate="user_agreements_download_pdf_title" data-translate-values="{ t0: agreement.name }"></span>
+                           <span class="fa fa-download" aria-hidden="true"></span>
```
Comment (JeremyDec) [inline client/app/account/user/agreements/user-agreements.html]: Same

File: client/app/account/user/contacts/user-contacts.html [hunk 7]
```diff
@@ -6,11 +6,12 @@ <h1 data-translate="useraccount_contacts"></h1>
...
             <div class="col-sm-3 text-right mt-4">
                 <a data-ng-if="!contactCtrl.loaders.init"
-                   class="btn btn-default"
+                   class="oui-button oui-button_secondary"
                    target="_blank"
                    rel="noopener"
                    data-ng-href="{{ ::contactCtrl.getAccountCreationUrl() }}"
-                   data-translate="useraccount_contacts_create_new_account">
+                   data-ng-click="contactCtrl.trackAccountCreation()">
+                   <span data-translate="useraccount_contacts_create_new_account"></span>
```
Comment (JeremyDec) [inline client/app/account/user/contacts/user-contacts.html]: Same

File: client/app/account/user/emails/user-emails.html [hunk 8]
```diff
@@ -9,7 +9,12 @@ <h1 data-translate="user_account_email_list_title"></h1>
         <p class="my-5">
             <span data-translate="user_account_email_info"></span><br />
             <span data-translate="user_account_email_assitance_info"></span>
-            <a href="#/ticket" data-translate="user_account_email_assitance_title"></a>
+            <a href="#/ticket"
+               data-track-name="account::emails::link::assistance_link"
+               data-track-on="click"
+               data-track-type="action">
+               <span data-translate="user_account_email_assitance_title"></span>
```
Comment (JeremyDec) [inline client/app/account/user/emails/user-emails.html]: Same

**Pull request merged**